src/main/scala/temple/builder/project/ProjectBuilder.scala

```diff
@@ -72,9 +72,11 @@ object ProjectBuilder {
           val rows             = MetricsBuilder.createDashboardRows(name, datasource, endpoints(service))
           val grafanaDashboard = GrafanaDashboardGenerator.generate(name.toLowerCase, name, rows)
           (File(s"grafana/provisioning/dashboards", s"${name.toLowerCase}.json"), grafanaDashboard)
-      } + (
+      } ++ Map(
```

Comment:
I think I might abstract this into the metrics builder so it's just one call that returns all the files. Will do in a follow up tho